src/test/java/showcase/neodymium/tests/random/RandomJobOffersTest.java

```diff
@@ -0,0 +1,46 @@
+package showcase.neodymium.tests.random;
+
+import static com.codeborne.selenide.Condition.exactText;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$$;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.codeborne.selenide.ElementsCollection;
+import com.xceptance.neodymium.util.NeodymiumRandom;
+
+import io.qameta.allure.Description;
+import showcase.flows.OpenPageFlow;
+
+public class RandomJobOffersTest
+{
+    @Test
```

Comment:
add a pre condition that no initial random value is set to ensure that the outcome of the test differs with every run.

Reply:
done

Reply:
Use something like the following otherwise configuring another initValue would not be detected by your pre conditions but still lead to the same result on different runs.
 
`Assert.assertNull("FixedRandomTest: neodymium.context.random.initialValue is set", Neodymium.configuration().initialRandomValue());`

Reply:
done